src/amo/components/App/styles.scss

```diff
@@ -67,6 +67,10 @@ body {
   max-width: $max-content-width;
 }
 
+.App-content-wrapper-homepage {
```

Comment:
This is a good point. I'll fix that.

Reply:
Hmm, even if I make the background of the navbar `#20123a`, which is the same as the start colour of the gradient, it still doesn't blend properly. The navbar is still significantly darker than the top of the hero element:

![Screenshot 2019-09-26 14 12 26](https://user-images.githubusercontent.com/142755/65713713-c00f6b00-e067-11e9-9045-70cd756c64cb.png)

Reply:
It looks like it's doing something similar to the picture at https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient#Composition_of_a_linear_gradient, but I even tried explicitly using `180deg` in the gradient, to say I want it straight down, and the colours still don't match up.

Reply:
Really? Which version of Firefox? When I edit the background colors in an inspector (Firefox release), it blends fine to my eye:

<details>
<summary>Screenshot</summary>

<img width="1390" alt="Screenshot 2019-09-26 14 12 55" src="https://user-images.githubusercontent.com/55398/65718361-fbf70000-e068-11e9-99a0-435d59bbfdb8.png">

</details>

If you really think it's a problem, you could make the top part darker for longer so that it blends more, like this:

```diff
diff --git a/src/amo/components/HeroRecommendation/styles.scss b/src/amo/components/HeroRecommendation/styles.scss
index 1fcfd0586..31bae95df 100644
--- a/src/amo/components/HeroRecommendation/styles.scss
+++ b/src/amo/components/HeroRecommendation/styles.scss
@@ -136,7 +136,7 @@
 }
 
 @mixin gradientStyle($startColor, $stopColor) {
-  background-image: linear-gradient($startColor, $stopColor);
+  background-image: linear-gradient($startColor 5%, 55%, $stopColor 80%);
 
   .HeroRecommendation-link {
     &:hover,

```

<details>
<summary>Screenshot</summary>

<img width="1390" alt="Screenshot 2019-09-26 14 23 21" src="https://user-images.githubusercontent.com/55398/65718529-542e0200-e069-11e9-866a-45050a376073.png">


</details>

Reply:
I'm on dev edition. I can try again with Nightly and Release. Which version are you using?

Reply:
Hmm, I tried it on Release and it does look a bit better. I'll push the change to this patch.

@kumar303 Are you still working on your review of this, or was this your only concern?

Reply:
That was my only comment thus far. I'll take another look.